copylot/hardware/cameras/basler/camera.py

```diff
@@ -0,0 +1,186 @@
+from copylot.hardware.cameras.abstract_camera import AbstractCamera
+from pypylon import pylon as py
+from pypylon import genicam
+import sys
+from copylot import logger
+import numpy as np
+from pdb import set_trace as st
+
+exitcode = 0
+
+
+class BaslerCameraException(Exception):
+    pass
+
+
+class BaslerCamera(AbstractCamera):
+
+    def __init__(self, camera_index: int = 0):
+        self.camera_index = camera_index
+        # get transport layer and all attached devices
+        self.camera = None
+        self.tl_factory = py.TlFactory.GetInstance()
+        self.devices = self.tl_factory.EnumerateDevices()
+        if len(self.devices) == 0:
+            raise py.RuntimeException("No camera connected")
+        else:
+            self.maxCameraToUSe = len(self.devices)
+            logger.info(str(self.maxCameraToUSe))
+            for device in self.devices:
+                logger.info(device.GetFriendlyName())  # return readable name
+
+    @property
+    def exposure_mode(self):
+        self.camera.ExposureMode.GetSymbolics()
+        self.camera.ExposureMode.GetValue()
```

Comment:
you gotta return something here @YangLiujames , I got the value but what does GetSymbolics return here?

Reply:
The Symbolics return all the available options. For this settings, I changed this function to fix this converns